test/ffmpeg-qsv/util.py

```diff
@@ -38,6 +38,7 @@ def get_supported_format_map():
     "I420"  : "yuv420p",
     "NV12"  : "nv12",
     "P010"  : "p010le",
+    "P012"  : "p012",
```

Comment:
Does ffmpeg support this format, yet?  If not, are there patches submitted to ffmpeg?  Patch links?

Reply:
Yes, the ffmpeg cartwheel supported this feature
https://github.com/intel-media-ci/ffmpeg-cartwheel/blob/master/patchset/0053-lavc-qsv-support-hevc-vp9-12bit.patch

Reply:
ok thanks. LGTM